.gitignore

```diff
@@ -19,7 +19,6 @@ package-lock.json
 package.json
 
 #amplify-do-not-edit-begin
-amplify/
```

Comment:
This line should be here - make sure you are fully up to date with master as this was added in recently.